.travis.yml

```diff
@@ -47,8 +47,14 @@ jobs:
       before_deploy:
         - docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
       deploy:
-        provider: script
-        on:
-          tags: true
-        script: /bin/bash scripts/docker_push.sh "${TRAVIS_TAG}" latest
+        - provider: script
+          on:
+            tags: true
+          script: /bin/bash scripts/docker_push.sh "${TRAVIS_TAG}" latest
+        - provider: pypi
+          user: ${test_pypi_USER}
+          password: ${test_pypi_PASS}
+          on:
+            branch: master
```

Comment:
We will want to build the package when a tag (=release) is created, not for every push to master :)